fix: Fixed the design of 'Tab List'

zvolcsey opened the pull request:
Hi! I fixed the design of 'Tab list'. I added 8px gap and changed the padding to 0.

Closes #1486

lucasbordeau commented:
Hi @zvolcsey thank you very much for your first contribution ! 

Could you provide a screenshot of your working solution so I can review more rapidly ?

lucasbordeau commented:
Also please feel free to join our Discord server with other contributors and where we are reactive every day : https://discord.gg/rYvMveY2

zvolcsey commented:
Hi @lucasbordeau! I made some screenshoots of my solution. I changed the gap value from 1 (4px) to 2 (8px) and changed the padding-left and padding-right from 1 to 0.

![code](https://github.com/twentyhq/twenty/assets/41576384/75293a0c-13a1-4888-994b-c0cc6974828c)
old:
![old](https://github.com/twentyhq/twenty/assets/41576384/7d429d6a-6bde-41b1-81c7-8c4f3bd7dcaa)
new:
![new](https://github.com/twentyhq/twenty/assets/41576384/f5fae737-fb22-4a1f-bf51-a25f6ee10899)

lucasbordeau commented on front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx:
```diff
@@ -32,10 +32,10 @@ const StyledTabListContainer = styled.div`
   height: 40px;
-  padding-left: ${({ theme }) => theme.spacing(1)};
-  padding-right: ${({ theme }) => theme.spacing(1)};
+  padding-left: ${({ theme }) => theme.spacing(0)};
```
I think you can go with `padding: 0` or remove the padding if there's no inherited padding maybe ?

zvolcsey commented:
Hi @lucasbordeau! There are updates in the issue. In the latest commit, I made the following improvements:

`ShowPageRightContainer.tsx`
- remove the padding
- I changed the `align-items` from `end` to `center`. => The bottom border is on the gray line.
old:
![tablist-old](https://github.com/twentyhq/twenty/assets/41576384/a91aff24-e535-4e00-a2bc-a970ae2a55df)
new:
![tablist-new](https://github.com/twentyhq/twenty/assets/41576384/0decaf8f-3c1e-4c4f-87a1-4cb31341e1de)

`Task.tsx`
- I added `gap` of 8px and `margin-left` 8px.

`TopBar.tsx`
- remove padding
old:
![task-old](https://github.com/twentyhq/twenty/assets/41576384/4c09cf12-0ab8-4e11-b08a-a9c83f723ad7)
new:
![task-new](https://github.com/twentyhq/twenty/assets/41576384/e51142e2-6fe2-4649-9d4d-87e909638f48)

`Tab.tsx`
- The background change from `tertiary` to `quaternary` while the tab is being pressed.
![click](https://github.com/twentyhq/twenty/assets/41576384/3931df9c-4b89-428a-a781-67814981706e)

charlesBochet reviewed (approved):
Thank you @zvolcsey. I'm merging it and I'll complete tests in a follow up PR
